### PR TITLE
Responsive Navbar added

### PR DESCRIPTION
**Pull request by arkumari2000**
I have added the responsive navbar, but there are 2 bugs on which I am working.
1. The hamburger button not changing to cross as it should after opening the navbar.
2. The right navbar is not visible in the mobile view.

I am working on these bugs and I will fix it soon.

### CODE REVIEW

**Comment by daltonfury42** [comment]
Can you pull latest new-frontend into your branch and resolve the conflicts? It would be mostly formatting changes.

**Comment by daltonfury42** [comment]
Check if bug 2 goes away after merging latest code. Refer to this PR, I had to do this to make the CSS work (open/close)

https://github.com/SimplQ/simplQ-frontend/pull/261/files

**Comment by arkumari2000** [comment]
@daltonfury42 I am trying to pull the lastest new-frontend but it is not updating. I have tried two methods.
Should I do all the changes manually first?
Screenshot from 2020-09-28 09-38-39

**Comment by daltonfury42** [comment]
You have what is called a "merge conflict". It happens when you want to merge branches (in this case our new-frontend branch into your branch), and in some files both the branches have changes, usually on same lines. Git is confused about which change to pick, and then this requires manual intervention.

In your screenshot git reports 6 such files, so you have to manually edit the files, and conclude the merge. Here is a nice video I found on youtube: https://www.youtube.com/watch?v=JtIX3HJKwfo

**Comment by arkumari2000** [comment]
Okay, Thanks for this. I will change manually and then merge.

**Comment by arkumari2000** [comment]
Bug 2 is resolved.
Bug 1 is due to this is not working. (This &[open] is not detecting whether the div is open or not).
Screenshot from 2020-09-29 06-21-10
I am working on it to find the solution.

**Review by daltonfury42 — COMMENTED** [review]
Here is the preview branch for till now: https://arkumari2.simplq.me/

On mobile, the sidebar with the fixed px width will overflow the screen. Let's define the width in rem to see if that's taken care of.

Also, we would need the sidebar to collapse on clicking anywhere outside, this we can raise an issue and move on. You can then take it up in a follow up PR or leave it for someone else to fix.

**Comment by daltonfury42** [comment]
> Bug 2 is resolved.
> Bug 1 is due to this is not working. (This &[open] is not detecting whether the div is open or not).
> Screenshot from 2020-09-29 06-21-10
> I am working on it to find the solution.

I'm not an expert, but is it because we are not setting `open` on the divs here? 
https://github.com/arkumari2000/simplQ-frontend/blob/new-frontend/simplq/src/components/pages/Home/Nav/Burger.jsx#L11

File: simplq/src/styles/homePage.module.scss
Comment (arkumari2000) [inline simplq/src/styles/homePage.module.scss]: see the width is already in rem i.e 25rem.

File: simplq/package.json
Comment (arkumari2000) [reply simplq/package.json:18]: Okay, I will remove.

File: simplq/src/components/pages/Home/Nav/Navbar.jsx
Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx:8]: I will add.

**Comment by arkumari2000** [comment]
> > Bug 2 is resolved.
> > Bug 1 is due to this is not working. (This &[open] is not detecting whether the div is open or not).
> > Screenshot from 2020-09-29 06-21-10
> > I am working on it to find the solution.
> 
> I'm not an expert, but is it because we are not setting `open` on the divs here?
> https://github.com/arkumari2000/simplQ-frontend/blob/new-frontend/simplq/src/components/pages/Home/Nav/Burger.jsx#L11

I will try to fix this if this is the bug.

**Comment by arkumari2000** [comment]
I have made the changes and now the navbar is working properly on the mobile screen also.
Please review.

File: simplq/src/components/pages/Home/Nav/Burger.jsx
Comment (daltonfury42) [inline simplq/src/components/pages/Home/Nav/Burger.jsx:11]: If we don't require `onClick={() => setOpen(!open)}` on these divs lets remove. As the parent already has an onclick defined

File: simplq/src/styles/homePage.module.scss
Comment (daltonfury42) [inline simplq/src/styles/homePage.module.scss:202]: here and other places, can we switch to `rem`s if possible?

**Review by daltonfury42 — COMMENTED** [review]
Almost there. This is great work!!

File: simplq/src/components/pages/Home/Nav/Burger.jsx
Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Burger.jsx:11]: with the parent, that was not detecting the onclick function. That is why I added  onClick={() => setOpen(!open)} to the child also.

File: simplq/src/styles/homePage.module.scss
Comment (arkumari2000) [reply simplq/src/styles/homePage.module.scss:202]: You want screen size to be in rems? But the standard media queries are preferred to be written in px.
But if you want in rem, I will change it.

Comment (daltonfury42) [reply simplq/src/styles/homePage.module.scss:202]: I see. Let me read up, but for now, let this stay.

**Comment by maaverik** [comment]
This looks great so far. Good job @arkumari2000. Do we need any more changes before merging @daltonfury42?

**Comment by daltonfury42** [comment]
Yeah, minor changes according to the comments above, and I will approve. Then I'll wait for your approval

**Comment by arkumari2000** [comment]
@maaverik Thank you. @daltonfury42 what are other changes that have to be done except px(which I don't have to change right now, as you said)?

**Comment by daltonfury42** [comment]
Remove firebase from `package.json` and add `alt` for the `img` on the navbar

**Review by daltonfury42 — APPROVED** [review]
@maaverik  after the above two changes, you can merge after reviewing.

**Comment by arkumari2000** [comment]
> Remove firebase from `package.json` and add `alt` for the `img` on the navbar

I have already done these changes. Please check.

**Comment by maaverik** [comment]
Can you check again? The diff shows these changes haven't been made. Did you push the latest version here?

Screenshot from 2020-10-01 22-41-17
Screenshot from 2020-10-01 22-40-42

File: simplq/package.json
Comment (arkumari2000) [inline simplq/package.json]: See, I have pushed this change.

File: simplq/src/components/pages/Home/Nav/Navbar.jsx
Comment (arkumari2000) [inline simplq/src/components/pages/Home/Nav/Navbar.jsx]: And this change too.

Comment (maaverik) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx]: I'm not sure what the problem here is, but I just pulled your latest code and the changes are not there.
Screenshot from 2020-10-01 22-55-05


Screenshot from 2020-10-01 22-54-46

And the diff you're referencing above seems to be outdated.
Screenshot from 2020-10-01 22-54-03

Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx]: What is the problem then?
let me push once more, maybe changes will reflect then.

Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx]: From my side, it is showing nothing to commit, and in my files, these changes are already done.
Screenshot from 2020-10-01 22-59-37
Screenshot from 2020-10-01 23-01-09
Screenshot from 2020-10-01 23-01-26

Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx]: The other change I have done i.e width:15rem before this push is reflecting but why not these?

Comment (maaverik) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx]: I'm looking at your github page and the changes aren't there either https://github.com/arkumari2000/simplQ-frontend/blob/new-frontend/simplq/src/components/pages/Home/Nav/Navbar.jsx. Try pulling from here first, then check and  push. I think it's because @daltonfury42 merged some changes to it which may have accidentally reverted these changes you made.

Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx]: Okay, I will try to do this. maybe things will work fine then.

Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx:8]: Done.
Please check now.

Comment (maaverik) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx:8]: Okay, looks like we're good now. Was that the issue?

Comment (arkumari2000) [reply simplq/src/components/pages/Home/Nav/Navbar.jsx:8]: Yes, I pulled first and then had to make the changes again.
maybe due to merge this issue came.

**Pull request merged**
